class-mb-rest-api.php

```diff
@@ -332,4 +339,17 @@ private function normalize_media_value( $field, $value ) {
 
 		return $value;
 	}
+
+	private function check_field_exists( $field_id, $field_value ) {
+		if ( empty( $field_value ) ) {
```

Comment:
Biến `$field_value` ở đây đúng ra phải là `$field` chứ?

Và dùng early return đi, sao để cả block `if` to thế kia.

Dùng thống nhất `echo` thay cho `print` nhé.

Phần dịch `__()` dùng `sprintf` chứ như kia ai mà translate được.